component/Header.tsx

```diff
@@ -1,12 +1,16 @@
 import Image from 'next/image'
 import React from 'react'
 import getConfig from 'next/config'
-import { useRouter } from 'next/router'
+import { BetterSocialEventTracking } from 'analytics/analyticsEventTracking'
+import { sendAnalytics } from '@services/analytics/analyticsServices'
 
 const { publicRuntimeConfig } = getConfig()
 
 const Header = ({fixedPosition = true}) => {
-    const toDownloadLink = () => window.open(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL, '_blank')
+    const toDownloadLink = () => {
+        sendAnalytics(BetterSocialEventTracking.PROFILE_SCREEN_DOWNLOAD_APP_BUTTON_CLICKED)
+        window.open(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL, '_blank')
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `toDownloadLink` function is now sending an analytics event before opening the download link. This is a good use of the new analytics functionality, but there's no error handling around the `sendAnalytics` call. If it fails for any reason (network issues, Segment down, etc.), this could prevent the download link from opening. Consider wrapping the `sendAnalytics` call in a try-catch block to ensure that errors are handled gracefully and don't affect the main functionality.
```diff
    const toDownloadLink = () => {
        try {
+            sendAnalytics(BetterSocialEventTracking.PROFILE_SCREEN_DOWNLOAD_APP_BUTTON_CLICKED)
+        } catch (error) {
+            console.error('Failed to send analytics:', error);
+        }
        window.open(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL, '_blank')
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->